doc/rst/developer/bestPractices/StandardModuleStyle.rst

```diff
@@ -0,0 +1,67 @@
+Standard Module Style
+=====================
+
+This document describes general guidance for style when writing a
+standard module. Of course there will be exceptions to this guidance when
+there is a good reason.
+
+Background
+----------
+
+Chapel does not use a separate namespace for types vs. module names vs
+local variables. As a result, it's generally a goal to avoid using the
+same name for, say, a type and a module.
```

Comment:
This doesn't really seem like "background" to me.  More like a note on module naming?  I'm also not sure what using a separate namespace for local variables would mean.